src/pudl/output/pudltabl.py

```diff
@@ -802,14 +807,40 @@ def gen_original_eia923(self, update=False):
             )
         return self._dfs["gen_og_eia923"]
 
+    def gen_fuel_allocated_generator_fuel_type_eia923(self, update=False):
```

Comment:
Is there fuel consumption also reported in the generation fuel table? What happens to that data in the allocation process? The name makes it sound like maybe that gets allocated too.